Makefile.js

```diff
@@ -648,8 +668,20 @@ target.gensite = function(prereleaseVersion) {
             let text = cat(filename),
                 title;
 
-            // 5. Prepend page title and layout variables at the top of rules
+            // 6. Prepend page title and layout variables at the top of rules
             if (path.dirname(filename).indexOf("rules") >= 0) {
+
+                // Find out if the rule requires a special docs portion (e.g. if it is recommended and/or fixable)
+                const isRecommended = recommended.indexOf(ruleName) >= 0;
+                const isFixable = fixable.indexOf(ruleName) >= 0;
```

Comment:
It doesn't really matter since there are a fairly small number of rules, but it might be better to use a `Set` for `recommended` and `fixable` since we don't care about the ordering here.